src/ops/view.rs

```diff
@@ -265,6 +281,22 @@ pub(super) fn pretty_view(
             .map(|v| v.to_string())
             .unwrap_or_else(|| format!("{warn}unknown{warn:#}"))
     )?;
+    if !metadata.categories.is_empty() {
+        if is_crate_from_crates_io {
+            writeln!(
+                stdout,
+                "{header}categories:{header:#} {note}{categories}{note:#}",
+                categories = metadata
+                    .categories
+                    .iter()
+                    .map(|category| Hyperlink::Category(category).to_string())
+                    .collect::<Vec<_>>()
+                    .join(" ")
+            )?;
+        } else {
+            write!(stdout, " {}", metadata.categories.join(" "))?;
+        }
+    }
```

Comment:
This should be a separate PR (ideally it would have started as an Issue) as this needs more design conversation (and tests)

Reply:
Okay~ I will revert it~

Reply:
Hi Ed~

I had reverted it 😉~